README.md

```diff
@@ -1,73 +1,97 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Please use the [Microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md) and change it based on your project 👍🏼